src/cls/IPM/Main.cls

```diff
@@ -1102,7 +1103,14 @@ ClassMethod Namespace(ByRef pCommandInfo) [ Internal ]
         do ##class(%Library.Prompt).GetString(prompt, .value)
         quit:value=""
         try {
-            set $namespace = $select($data(list(value), ns): $listget(ns), 1: value)
+            set selectedNamespace = $select($data(list(value), ns): $listget(ns), 1: value)
+            if '..IsIPMEnabled(selectedNamespace) {
+                write $$$FormattedLine($$$Red," IPM is not enabled in this namespace.")
+                hang 0.5
+                write $char(13),*27,"[K",*27,"[A"
```

Comment:
Please add a comment if writing out unicode chars. We have the $$$FormattedLine and other macros that add semantic meaning to the chars so see if any of those suffice/add to them (I notice the same pattern is used below so worth fixing it there too)